common/src/main/java/tachyon/security/UserInformation.java

```diff
@@ -107,7 +115,81 @@ static String getOsPrincipalClassName() {
     }
   }
 
-  // TODO: TACHYON-602 - add TACHYON_LOGIN_MODULE to convert an OS user to Tachyon user.
+  /**
+   * A login module that search the Kerberos or OS user, and then convert to a Tachyon user.
+   */
+  public static class TachyonLoginModule implements LoginModule {
+    private Subject mSubject;
+
+    @Override
+    public boolean abort() throws LoginException {
+      return true;
+    }
+
+    @Override
+    public void initialize(Subject subject, CallbackHandler callbackHandler,
+                           Map<String, ?> sharedState, Map<String, ?> options) {
+      mSubject = subject;
+    }
+
+    @Override
+    public boolean login() throws LoginException {
+      return true;
```

Comment:
Can you add some `TODO:` for things that are not pushed yet? It will make code review much easier!